test/test_point.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+
+from src.point import Point
+
+
+class TestCircle(unittest.TestCase):
```

Comment:
`Point`, not circle.

Kinda aside, it is more typical that test class names end with `Test` (ie, testing `ClassName` in `ClassNameTest`), rather than start with them. `unittest` doesn't actually care, you're free to define your own style. I've also seen the prefix style used (usually in older projects), but my preference is the suffix, since it's more like how you'd just speak about it. X tests are in the XTest class, rather than TestX class.

Reply:
[I went with what `unittest` showed](https://docs.python.org/3/library/unittest.html#basic-example), but I am not particularly committed to either way.

Reply:
(╯°□°)╯︵ ┻━┻

Reply:
I suppose an argument for keeping it as `TestX` is the naming convention of the file names themselves, as they have `test_` at the beginning of file names.

Reply:
I'd say that's an argument for `XTest`. Functions are actions -- `test_this_thing`. Namespaces are nouns -- `ThingTest`.